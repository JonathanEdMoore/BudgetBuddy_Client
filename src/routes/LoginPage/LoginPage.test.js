import React from 'react'
import ReactDOM from 'react-dom'
import LoginPage from './LoginPage'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'


describe('LoginPage smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><LoginPage /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('LoginPage snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<LoginPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})