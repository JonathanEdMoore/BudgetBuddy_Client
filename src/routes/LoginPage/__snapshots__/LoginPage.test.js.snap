// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage snapshot test renders the UI as expected 1`] = `
<div>
  <main
    role="main"
  >
    <header>
      <MainHeader />
      <h2>
        Log In
      </h2>
    </header>
    <section>
      <LoginForm
        onLoginSuccess={[Function]}
      />
    </section>
  </main>
</div>
`;
